modules/webbrowser/src/processors/webbrowserprocessor.cpp

```diff
@@ -166,52 +155,6 @@ WebBrowserProcessor::WebBrowserProcessor()
     cefInteractionHandler_.setHost(browser_->GetHost());
     cefInteractionHandler_.setRenderHandler(renderHandler_);
     addInteractionHandler(&cefInteractionHandler_);
-
```

Comment:
I think the option to add additional properties is quite nice. There should be at least one processor where we can add properties at runtime.

Reply:
Python processor :)

Reply:
This approach (setting properties directly) is more flexible, less complicated and easier to use according to me